[ANDROID] react-native-webview:compileDebugJavaWithJavac BUILD ERROR on 10.2.1

**Bug description:**
Hello, 
just upgraded to 10.2.1 from 9.4. 

I am getting following two errors when building android. 

```
.../node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java:1239: error: method setWebChromeClient(WebChromeClient) is already defined in class RNCWebView
    public void setWebChromeClient(WebChromeClient client) {
                ^


.../node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java:276: error: cannot find symbol
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
                                                    ^
  symbol:   variable Q
  location: class VERSION_CODES
```

I have done ./gradlew clean and reset npm cache, but same result. 

**Environment:**
 - OS: MacOS
 - OS version: 10.15.5
 - react-native version: 0.61.5
 - react-native-webview version: 10.2.1

--- a/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
+++ b/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java
@@ -409,7 +409,7 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
   public void setMessagingModuleName(WebView view, String moduleName) {
     ((RNCWebView) view).setMessagingModuleName(moduleName);
   }
-   
+
   @ReactProp(name = "incognito")
   public void setIncognito(WebView view, boolean enabled) {
     // Remove all previous cookies
@@ -660,7 +660,7 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
         public Bitmap getDefaultVideoPoster() {
           return Bitmap.createBitmap(50, 50, Bitmap.Config.ARGB_8888);
         }
-        
+
         @Override
         public void onShowCustomView(View view, CustomViewCallback callback) {
           if (mVideoView != null) {
@@ -1086,8 +1086,10 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
       }
     }
 
+    WebChromeClient mWebChromeClient;
     @Override
     public void setWebChromeClient(WebChromeClient client) {
+      this.mWebChromeClient = client;
       super.setWebChromeClient(client);
       if (client instanceof RNCWebChromeClient) {
         ((RNCWebChromeClient) client).setProgressChangedFilter(progressChangedFilter);
@@ -1234,16 +1236,9 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
       destroy();
     }
 
-    WebChromeClient mWebChromeClient;
-    @Override
-    public void setWebChromeClient(WebChromeClient client) {
-      this.mWebChromeClient = client;
-      super.setWebChromeClient(client);
-    }
-
     @Override
     public void destroy() {
-      if(mWebChromeClient!=null){
+      if (mWebChromeClient != null) {
         mWebChromeClient.onHideCustomView();
       }
       super.destroy();

